IONOS/src/components/UserMenu.svelte

```diff
@@ -0,0 +1,195 @@
+<svelte:options customElement="ionos-user-menu" />
+
+<script>
```

Comment:
suggestion: using typescript

We should use the advantages typescript gives us regarding code quality and safety
```suggestion
<script lang="ts">
```